src/main/java/fr/insee/aventcalendar/model/leaderboard/UsersProfile.java

```diff
@@ -0,0 +1,15 @@
+package fr.insee.aventcalendar.model.leaderboard;
+
+import java.util.Map;
+
+public class UsersProfile {
```

Comment:
Not sure if it should be `UsersProfile` or `UserProfiles`

Reply:
Every user has his own profile, and there is several users, so...